src/app/shared/custom-components/logo/logo.component.html

```diff
@@ -0,0 +1,2 @@
+<div class="logo"></div>
+<!-- <mat-icon svgIcon="logo"></mat-icon> -->
```

Comment:
we need the logo to be a link. maybe try something like this? and no background and @media

```
<a href="">
    <picture>
        <source srcset="img/icon-mobile.svg" media="(max-width: 768px)" >
        <img src="img/icon-desktop.svg" alt="">
    </picture>
</a>
```

Reply:
Removed logo component. Made link. But I left image loading as background.